docs/agents/sematext-agent/permission-requirements.md

```diff
@@ -0,0 +1,20 @@
+title: Permission requirements
+description: Why Sematext Agent needs access to certain host files and directories
+
+Sematext Agent requires root privileges to perform several operations:
+
+- scan active processes as well as their resources such as file descriptors, open sockets, etc.
+- load eBPF probes on kernel symbols
+- monitoring network traffic
+- accessing process log files
+- opening the `/dev/kmsg` device for reading kernel log buffer messages
+
+## Container setups
+
+In container setups, Sematext Agent requires that the following bind mounts are defined. Please note, the majority of bind mounts are propagated to container as read-only volumes.
+
+- `/:/hostfs:ro` - the root partition is mounted in order to collect disk space metrics from inside container.
```

Comment:
space metrics from inside container
=>
space metrics from inside the container